app/src/main/java/nl/tudelft/bbw/controller/KeyReader.java

```diff
@@ -123,4 +124,16 @@ private static EdDSAPublicKey convertToPublicKey(byte[] encodedPublicKey) throws
         X509EncodedKeySpec encoded = new X509EncodedKeySpec(encodedPublicKey);
         return new EdDSAPublicKey(encoded);
     }
+
+    /**
+     * decryptFile method
```

Comment:
It makes sure you do not write the same code multiple times and it decreases the amount of lines per method.